docs/cli-arguments.md

```diff
@@ -0,0 +1,90 @@
+# Command line arguments
+
+kube-state-metrics can be configured through command line arguments.
+
+Those arguments can be passed during startup when running locally:
+
+`kube-state-metrics --telemetry-port=8081 --kubeconfig=<KUBE-CONFIG> --apiserver=<APISERVER> ...`
+
+Or configured in the `args` section of your deployment configuration in a Kubernetes / Openshift context:
+
+```yaml
+spec:
+  template:
+    spec:
+      containers:
+        - args:
+          - '--telemetry-port=8081'
+          - '--kubeconfig=<KUBE-CONFIG>'
+          - '--apiserver=<APISERVER>'
+```
+
+## Available options:
```

Comment:
this should be easy to generate, could you add some automation that would automate this? We do this in other projects already like: 

* https://github.com/brancz/kube-rbac-proxy/blob/master/scripts/generate-help-txt.sh
* and then template with [embedmd](https://github.com/campoy/embedmd) (see the raw of this readme): https://github.com/brancz/kube-rbac-proxy/blob/master/README.md#usage

You can pretty much copy this I think: https://github.com/brancz/kube-rbac-proxy/blob/e31509ba63845ac2afdd26afef28dafa60221d26/Makefile#L53-L55

Reply:
On it!

Reply:
Done ! Let me know if it's ok for you: I added "scripts/" folder with the sh script + embedmd + new `make generate` that relies on `make embedmd` and `make build-local`.